tools/azsdk-cli/docs/specs/0-scenario-1.spec.md

```diff
@@ -263,14 +263,13 @@ Update the package metadata for Health Deidentification SDKs to prepare for a pr
 
 **Prompt:**
 ```
-Run all PR checks locally for the Health Deidentification SDKs before I create pull requests.
+Run all validation checks locally for the Health Deidentification SDKs before I create pull requests.
```

Comment:
nit: Given we wanted to update `all` checks I would be consistent with that terminology.